doc/source/user_guide/io.rst

```diff
@@ -5593,164 +5593,166 @@ Given the next test set:
 
 .. code-block:: python
 
-   from numpy.random import randn
+    from numpy.random import randn
```

Comment:
can you change this example to use our formatting, meaning don import like this, rather use
``np.random.randn`` directory, and ``np.random.seed``

Reply:
Done